### PR TITLE
Make fixture init event order explicit

### LINKED ISSUE #2907: Physics relies on unguaranteed local EventBus ordering
`CollisionWakeSystem` and `FixtureSystem` both subscribe to `PhysicsInitializedEvent` (the former with a directed sub). These two systems implicitly rely on `CollisionWakeSystem` getting fired *first* due to implementation details in EventBus, despite not having any explicit ordering specified. If `CollisionWakeSystem` fires second then non-colliding fixtures get into broadphase which explodes *very violently*.

### PR DESCRIPTION
**Pull request by wrexbe**
Closes https://github.com/space-wizards/RobustToolbox/issues/2907

### CODE REVIEW

File: Robust.Shared/Physics/FixtureSystem.cs [hunk 1]
```diff
@@ -29,7 +29,7 @@ public override void Initialize()
             SubscribeLocalEvent<FixturesComponent, ComponentGetState>(OnGetState);
             SubscribeLocalEvent<FixturesComponent, ComponentHandleState>(OnHandleState);
 
-            SubscribeLocalEvent<PhysicsInitializedEvent>(OnPhysicsInit);
+            SubscribeLocalEvent<PhysicsInitializedEvent>(OnPhysicsInit, after: new[] { typeof(CollisionWakeSystem) });
```
Comment (metalgearsloth) [inline Robust.Shared/Physics/FixtureSystem.cs]: Probably better off just explicitly calling it rather than using ordered events tbh.

Comment (wrexbe) [reply Robust.Shared/Physics/FixtureSystem.cs]: I think I can do that... give me a bit

**Comment by wrexbe** [comment]
Is this what you had in mind?

File: Robust.Server/GameObjects/EntitySystems/PhysicsSystem.cs [hunk 2]
```diff
@@ -22,6 +24,18 @@ public override void Initialize()
...
+        protected override void OnPhysicsInitialized(EntityUid uid)
+        {
+            if (EntityManager.TryGetComponent(uid, out CollisionWakeComponent? wakeComp))
+            {
+                _collisionWakeSystem.OnPhysicsInit(uid, wakeComp);
+            }
+            if (EntityManager.TryGetComponent(uid, out FixturesComponent? fixtureComp))
+            {
+                _fixtureSystem.OnPhysicsInit(uid, fixtureComp);
+            }
```
Comment (metalgearsloth) [inline Robust.Server/GameObjects/EntitySystems/PhysicsSystem.cs]: Client should also be doing this, rest is probably okay.

**Pull request merged**